rules/aip0191/aip0191.go

```diff
@@ -45,3 +45,4 @@ func hasPackage(f *desc.FileDescriptor) bool {
 }
 
 var versionRegexp = regexp.MustCompile("^v[0-9]+(p[0-9]+)?((alpha|beta)[0-9]*)?$")
+var validCharacterRegexp = regexp.MustCompile("^[a-zA-Z 0-9\\_\\/]*$")
```

Comment:
```suggestion
var validCharacterRegexp = regexp.MustCompile(`^[a-zA-Z 0-9\\_\\/]*$`)
```

I realize that the expression directly above uses `"` but I guess the golang linter wasn't run on that haha